ui-grid/ui-grid.d.ts

```diff
@@ -601,6 +601,13 @@ declare module uiGrid {
          */
         enableFiltering?: boolean;
         /**
+        * False by default. When enabled, this adds a settings icon in the top right of the grid, 
+        * which floats above the column header. The menu by default gives access to show/hide columns, 
+        * but can be customized to show additional actions.
+        * @default false
+        */
+        enableGridMenu?: boolean;
```

Comment:
I was able to find `useExternalFiltering` in the ui-grid API documentation (http://ui-grid.info/docs/#/api/ui.grid.class:GridOptions), but I couldn't find `enableGridMenu` documented anywhere.  I assume it does exist as I found it referred to in the source (https://github.com/angular-ui/ui-grid/blob/e8e5653/src/features/exporter/js/exporter.js#L1221), but just curious how you came about the documentation for that option.  Is this a case where it would be good to update and improve the ui-grid documentation as well?